files/sdw-admin.py

```diff
@@ -133,15 +133,29 @@ def perform_uninstall(keep_template_rpm=False):
     )
 
 
+def show_warning():
+    print(
+        "This script will provision the SecureDrop Workstation.\n\n"
+        "The SecureDrop Workstation is currently in a closed beta.\n"
+        "We do not recommend installing it in production independently until\n"
+        "the project has reached general availability.\n\n"
+        "If you are not part of the beta program, but are interested in joining,\n"
+        "please contact us at securedrop@freedom.press or via our support portal."
+    )
+    input("Press [Enter] to proceed…")
```

Comment:
I think this would keep our custom testing CI from running right now. I don't mind adding it but I think we'll need a `--skip-warning` argument along with this, and I guess changes in the CI as well?